src/ape/cli/options.py

```diff
@@ -99,3 +103,32 @@ def account_option_that_prompts_when_not_given():
         type=AccountAliasPromptChoice(),
         callback=_account_callback,
     )
+
+
+def _load_contracts(ctx, param, value) -> Optional[Union[ContractType, List[ContractType]]]:
+    if not value:
+        return None
+
+    if len(project.contracts) == 0:
+        raise ContractError("Project has no contracts.")
+
+    is_multiple = isinstance(value, (tuple, list))
```

Comment:
Wait, you don't opt-in to this behavior? Could make it harder to use

Reply:
You do opt in.
See the examples in the `How to verify it` section!

Reply:
When you pass in `multiple=True`, it works the exact same was when doing it for `click.option()`, you get a list as the `value`.

The default value is `False`, which makes `value` not a list but a single item.

Here, we detect if the user is doing multiple mode or not and return the same type of value (list or single item)

Reply:
I will add a comment

Reply:
It seems like it would be cleaner to access that kwarg somehow, but I could see how this is not available